src/classes/AnonymisePersonExtension.cls

```diff
@@ -46,7 +46,7 @@ public with sharing class AnonymisePersonExtension {
             System.debug('111111111111111111111111111111111 person.Consent_Status__c: '+person.Consent_Status__c);
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.